src/main/webapp/app/layouts/navbar/navbar.component.html

```diff
@@ -18,6 +18,14 @@
                     </span>
                 </a>
             </li>
+            <li class="nav-item" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">
```

Comment:
it is fairly awkward since i can view pools without logging. 

is that an expected behaviour?

Reply:
as far as I remember permissions to views is separate task, also we do not have logging view so you go straight forward to main page

Reply:
So its out of scope of this task then. :)